.github/workflows/publish.yml

```diff
@@ -4,6 +4,7 @@ on:
   push:
     branches:
     - 'main'
+    - '115-duplicate-entries-in-state_history-2'
```

Comment:
can you please elaborate why this change is required, or you using it while developing